prow/plugins/approve/approve.go

```diff
@@ -462,7 +463,17 @@ func handle(log *logrus.Entry, ghc githubClient, repo approvers.Repo, githubConf
 	start = time.Now()
 	notifications := filterComments(commentsFromIssueComments, notificationMatcher(botUserChecker))
 	latestNotification := getLast(notifications)
-	newMessage := updateNotification(githubConfig.LinkURL, opts.CommandHelpLink, opts.PrProcessLink, pr.org, pr.repo, pr.branch, latestNotification, approversHandler)
+
+	var testFreezeResult *testfreeze.Result
+	// Enable test freeze only for the k/k repo and master branch
+	if pr.org == "kubernetes" && pr.repo == "kubernetes" && pr.branch == "master" {
```

Comment:
I'm not sure how much I like hardcoding k/k logic into an approve plugin that is supposed to be generic, tbh. It does not seem to be too invasive though.